src/app/Plans/components/Wizard/MappingForm.tsx

```diff
@@ -61,6 +61,9 @@ const MappingForm: React.FunctionComponent<IMappingFormProps> = ({
     { source: null, target: null },
   ]);
 
+  // TODO add to disable Next step
+  const isEveryItemFilled = () => builderItems.every((item) => item.source && item.target);
```

Comment:
Ah, okay, we do use `isEveryItemFilled` in multiple places. I'd say either make both copies of it plain booleans or move it to a `helpers.ts` file somewhere and have it take `builderItems` as an argument. It's pretty trivial, so either way is fine with me, I don't think we lose much having it duplicated.

Reply:
Yes, I agree.